tests/unit/api/bucketPut.js

```diff
@@ -269,4 +269,29 @@ describe('bucketPut API', () => {
         });
         done();
     });
+
+    it('should pick up updated rest endpoint config', done => {
+        const bucketName = 'new-loc-bucket-name';
+        const newRestEndpoint = 'newly.defined.rest.endpoint';
+        const newLocation = 'scality-us-west-1';
```

Comment:
the new location would have to be added to the locationConfig and parsed by the location constraint parser to create a new in memory client.

Reply:
This new location (`'scality-us-west-1'`) is already present in the testing location config, the `bucketPut` call fails otherwise.